frontends/ol-components/src/components/Dialog/Dialog.tsx

```diff
@@ -115,6 +116,7 @@ const Dialog: React.FC<DialogProps> = ({
       disableEnforceFocus={disableEnforceFocus}
       PaperProps={PaperProps}
       TransitionComponent={Transition}
+      aria-labelledby={titleId}
```

Comment:
This was an issue I noticed while looking at accessibility stuff. Previously MUI did this for us, but when we stopped using their DialogTitle component, we lost the automatic id assignment.